Nighttime-Operations/Auxiliary-Telescope/AT-Shutdown/Shutdown.rst

```diff
@@ -8,10 +8,117 @@
+If there are no targets, the Scheduler will stop adding new ones.
+If you need to stop the Scheduler manually, you can use the ``auxtel/scheduler/stop.py`` script from ScriptQueue without any additional configurations.
+
+See more detailed descriptions on `Scheduler <https://obs-ops.lsst.io/Nighttime-Operations/Scheduler/night-time-operations/index.html>`__.
+
+
+Shutdown AuxTel
+^^^^^^^^^^^^^^^
+
+After stopping scheduler, you can shutdown AuxTel using ``axutel/shutdown.py`` script from ScriptQueue. 
+``shutdown.py`` script does not any configurations. This script includes:
+
+**1. Closing the M1 cover.**
+: First, disable ATAOS corrections. Then, close the M1 cover and vent gates.  
+If mirror is opened, ``shutdown`` script will not able to close the dome slit. 
+
+**2. Closing the Dome Shutter.**
+: Closing the dome shutter and dropout door if it is opened.  
+ 
+**3. Parking the Dome** 
+: The parking position is az=285.0 deg. Then, the dome following is disable after parking. 
+
+**4. Parking the Telescope**
+: The parking position is El = 80 deg, Az = 0.0 deg, rot = 0.0 deg for telescope.
+Tracking will then be stopped.  
+
+**5. Putting CSCs in Stanby.**
+: The AT CSC components ``ATMCS``, ``ATOS``, ``ATPtg``, ``ATDome``, ``ATDomeTrajector``, ``ATOS``, ``ATPneumatics``, ``ATHexapod``  will be change to Stanby. 
+
+After parking and shutdown for Auxillary Telescope are completed, you can check the status of Dome and Telescope on the LOVE. 
+
+- **Status of Telescope and Dome** 
+.. image:: ./_static/ATDome_park.png
+    :name: Auxiliary Telescope and Dome at the parking position
+
+- **Status of Mirrors and Mirror Covers**
+.. image:: ./_static/AT_LightPath.png
+   :scale: 50 %
+
+   
+Taking Daytime Calibration
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+After ``shutdown.py`` script is completed, run ``make_latiss_calibration.py`` with the following configuration.
+
+.. code-block:: yaml
+
+    n_bias: 50
+    n_dark: 22
+    exp_times_dark: [5, 5, 5, 5, 5, 15, 15, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30]
+    generate_calibrations: True
+    do_verify: True
+    certify_calib_begin_date: "2023-03-16" #change date to match end of observing night
+    certify_calib_end_date: "2023-03-17" #change date to correspond to 24-hours perios
+    script_mode: BIAS_DARK
+
+
+If it is not working, 
+
+1. Run ``make_image_latiss`` to take biases with the following configurations:
+
+.. code-block:: yaml
+
+    image_type: BIAS
+    nimages: 50
+    reason: 'dailyCals' 
+
+2. Run ``make_image_latiss`` to take dark images with the following configurations:
+
+.. code-block:: yaml
+    
+    image_type: DARK
+    nimages: 23
+    exp_times: [5, 5, 5, 5, 5, 15, 15, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30]
+    reason: "dailyCals"
+
+
+*Stanby_latiss*
+^^^^^^^^^^^^^^^
+After parking the AuxTel and taking daytime calibration, LATISS on CSC are still in Enabled.
```

Comment:
You can add this to the ScriptQueue to run after the make_latiss_calibrations.py script. It is not necessary that you stay and wait for the first script to finish.

Reply:
Typo: STANDBY_LATISS , d missing.